index.html

```diff
@@ -145,38 +138,33 @@ <h4>
                 </li>
                 <li>Let <var>url</var> be the result of running the
                 <a data-cite="!URL#concept-url-parser">URL parser</a> on <var>
-                  data</var>'s <a for="ShareData">url</a>, with
-                  <var>base</var>, and no <var>encoding override</var>.
+                  data</var>'s <a>url</a>, with <var>base</var>, and no
+                  <var>encoding override</var>.
                 </li>
                 <li>If <var>url</var> is failure, <a data-cite=
                 "!promises-guide#reject-promise">reject</a> <var>p</var> with
-                <a data-cite=
-                "!WEBIDL#exceptiondef-typeerror"><code>TypeError</code></a>,
-                and abort these steps.
+                <a>TypeError</a>, and abort these steps.
                 </li>
                 <li>Set <var>data</var> to a copy of <var>data</var>, with its
-                <a for="ShareData">url</a> field set to the result of running
-                the <a data-cite="!URL#concept-url-serializer">URL
-                serializer</a> on <var>url</var>.
+                <a>url</a> field set to the result of running the
+                  <a data-cite="!URL#concept-url-serializer">URL serializer</a>
+                  on <var>url</var>.
                 </li>
               </ol>
             </li>
             <li>If the method call was not <a data-cite=
             "!HTML#triggered-by-user-activation">triggered by user
             activation</a>, <a data-cite=
-            "!promises-guide#reject-promise">reject</a> <var>p</var> with
-            <a data-cite=
-            "!WEBIDL#notallowederror"><code>NotAllowedError</code></a>, and
-            abort these steps.
+            "!promises-guide#reject-promise">reject</a> <var>p</var> with a
+              "<a>NotAllowedError</a>" <a>DOMException</a>, and abort these steps.
             </li>
             <li>
               <a data-cite="!HTML#in-parallel">In parallel</a>:
               <ol>
                 <li>If there are no <a>share targets</a> available,
                 <a data-cite="!promises-guide#reject-promise">reject</a> <var>
-                  p</var> with <a data-cite=
-                  "!WEBIDL#aborterror"><code>AbortError</code></a>, and abort
-                  these steps.
+                  p</var> with "<a>AbortError</a>" <a>DOMException</a>, and
```

Comment:
Clarified that "AbortError" isn't an error class, it's a `DOMException` type. Same with "NotAllowedError".

Reply:
Cool. Can you be consistent with NotAllowedError and say "*an* `"AbortError"` `DOMException`" (i.e., add "an" in front of AbortError).